### PR TITLE
Code coverage task for coveralls

### PR DESCRIPTION
**Pull request by danquirk**
Testing code coverage task with Travis

Edit: not ready for checkin, but needed to submit a PR for testing purposes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
-        "jake": "latest",
-        "mocha": "latest",
-        "chai": "latest",
-        "browserify": "latest",
-        "istanbul": "latest"
-    },
-    "scripts": {
-        "test": "jake runtests"
-    }
+  "name": "typescript",
```
Comment (DanielRosenwasser) [inline package.json]: Any reason why we cared enough to go back to 2 spaces?

Comment (danquirk) [inline package.json]: No, I noticed that too, will fix before I check in if this system works. Either Sublime or VSCode must've auto-reformatted me.

**Comment by danquirk** [comment]
Apparently the build must also take < 10min which code coverage does not so Travis is failing due to that

**Comment by mhegazy** [comment]
@danquirk what is the status of this PR?

**Comment by danquirk** [comment]
Was playing with it recently and got pulled off to other work. Unfortunately the nature of the task is such that I can't really do much testing locally. I'll just close it for now though.

**Pull request closed without merging**